CMSgov/cmcs-eregulations
# EREGCSC-1538 [Spike] Highlight search text in regs

Resolves [EREGCSC-1538](https://jiraent.cms.gov/browse/EREGCSC-1538)

**Description:**
Spike to show basic implementation of highlighting text in regulations reader view when being linked from the Search Results page.

**This pull request changes:**

- Adds search param to reader view URL when coming from the search page
- highlights text in targeted section (via hash in URL) in a case-insensitive way using JS

**Steps to manually verify this change**

1. Go to `/search` and search for something
2. Follow a search result link into reg text reader view
3. See that text highlighted within the targeted section
